README.md

```diff
@@ -1,31 +1,53 @@
 # tket
 
+[![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://tketusers.slack.com/join/shared_invite/zt-18qmsamj9-UqQFVdkRzxnXCcKtcarLRA#/shared-invite/email)
+[![Stack Exchange](https://img.shields.io/badge/StackExchange-%23ffffff.svg?style=for-the-badge&logo=StackExchange)](https://quantumcomputing.stackexchange.com/search?q=user:19610+[pytket])
```

Comment:
Should `user:19610` be part of this URL? When I follow the link I see a message about "not deleted user 19610" near the top of the search results and I don't know what this means.

Reply:
Ah this link got changed somehow.

Using `https://quantumcomputing.stackexchange.com/tags/pytket` seems to work